src/utils/swagger.config.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  DocumentBuilder,
+  OpenAPIObject,
+  SwaggerCustomOptions,
+} from '@nestjs/swagger';
+
+export const swaggerConfig = (): Omit<OpenAPIObject, 'paths'> => {
+  return new DocumentBuilder()
+    .setTitle('Blood Donor API')
+    .setDescription('API documentation for the Blood Donor Mobile App')
```

Comment:
It's actually the API docs for the backend API, which is used by the mobile app ;)

Reply:
Done